server/views/pages/overview/partials/overviewTab/_actionsHeader.njk

```diff
@@ -6,23 +6,25 @@
     <h2 class="govuk-heading-m">Goals in progress</h2>
   </div>
 
-  {% if hasEditAuthority %}
+  {% if not featureToggles.goals.archiveEnabled %}
```

Comment:
Now we only show the original green "Add a new goal" button if the feature toggle is disabled